Source/Core/InputCommon/ControllerEmu/StickGate.h

```diff
@@ -15,6 +15,9 @@
 
 namespace ControllerEmu
 {
+// Minimum stick distance from the center before virtual notches are applied.
+constexpr auto MINIMUM_NOTCH_DISTANCE = 0.9;
```

Comment:
This should likely be:

```suggestion
constexpr ControlState MINIMUM_NOTCH_DISTANCE = 0.9;
```

Given it's used with values of that type.